include/oneapi/dnnl/dnnl_graph.hpp

```diff
@@ -819,6 +819,7 @@ class op : public op_handle {
         Reciprocal = dnnl_graph_op_reciprocal,
         ReduceL1 = dnnl_graph_op_reduce_l1,
         ReduceL2 = dnnl_graph_op_reduce_l2,
+        ReduceAMax = dnnl_graph_op_reduce_amax,
```

Comment:
This addition causes a failure in `tests/gtests/graph/api/test_cpp_api_op.cpp:CreateAllOps`. Please add this enum value at the appropriate point in this file.

Reply:
Did you encounter the following error ?
340: [ RUN      ] APIOp.CreateAllOps
340: [       OK ] APIOp.CreateAllOps (0 ms)
340: [ RUN      ] APIOp.CreateWithInputList
340: [       OK ] APIOp.CreateWithInputList (0 ms)
340: [ RUN      ] APIOp.CreateWithDefaultName
340: [       OK ] APIOp.CreateWithDefaultName (0 ms)
340: [ RUN      ] APIOp.SetInput
340: [       OK ] APIOp.SetInput (0 ms)
340: [ RUN      ] APIOp.SetOutput
340: [       OK ] APIOp.SetOutput (0 ms)
340: [ RUN      ] APIOp.SetAttr
340: [       OK ] APIOp.SetAttr (0 ms)
340: [ RUN      ] APIOp.ShallowCopy
340: [       OK ] APIOp.ShallowCopy (0 ms)
340: [ RUN      ] APIOp.SoftPlusAttr
340: unknown file: Failure
340: C++ exception with description "could not add op to the graph" thrown in the test body.

Reply:
No, this is the issue I saw:
>/Users/runner/work/oneDNN/oneDNN/oneDNN/tests/gtests/graph/api/test_cpp_api_op.cpp:125: Failure
Expected equality of these values:
  i
    Which is: 53
  static_cast<size_t>(kind)
    Which is: 54
[  FAILED  ] APIOp.CreateAllOps (0 ms)
[ RUN      ] APIOp.CreateWithInputList
[       OK ] APIOp.CreateWithInputList (0 ms)
...

CreateAllOps is failing because your added enum value shifts the value of the enums that come after it (53 -> 54). Your error looks like something else, but I'm not sure what.

Reply:
Thank you for explaining the cause.